mathml/relations/css-styling/dynamic-dir-1.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Test dynamically changing dir attribute</title></head>
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#cssproperties">
```

Comment:
I think you can add https://mathml-refresh.github.io/mathml-core/#dom too, since this test is about testing dynamic change.

Reply:
so, another link rel="help" you mean?

Reply:
Yes, I think other tests use multiple `<link rel="help">` tags